documentation/Cohen Hubbard Oesterwinter.tex

```diff
@@ -0,0 +1,166 @@
+
+\subsection*{Finite difference formulæ}
+In this section we prove a backward difference formula that makes it possible to compute an approximation of any derivative of a function on an equally-spaced grid at any desired order.  We then derive a corollary based on the second derivative $f\dersecond\of{x}$ that is useful for the following sections. 
+\begin{lemma}
+Given a sufficient regular function $f\of{x}$ and two integers $m ≤ n$, there exists a family of rational numbers $\gl^{m}_{n,\gn}$, with $0 ≤ \gn ≤ n$, such that:
+
+\begin{equation}
+f^{(m)}\of{x_0} = \frac{\pa{-1}^m}{h^m} \sum{\gn = 0}[n]{\gl^{m}_{n,\gn} f\of{x_0 - \gn h}} + \BigO\of{h^{n - m + 1}} 
+\label{fornbergbackward}
+\end{equation}
+Furthermore, the $\gl^{m}_{n,\gn}$ are independent from $x_0$ and $h$.
+
+\begin{proof}
+\cite{Fornberg1988} gives finite difference formulæ for any order and for kernels of arbitrary size.  Given a family $\VectorSymbol{\ga} = \pa{\ga_0, \ga_1, ..., \ga_N}$ of points (which may not be equidistant), the $m$-th derivative at any point $x_0$ may be approximated to order $n - m + 1$ as:
+\[
+f^{(m)}\of{x_0} ≅ \sum{\gn = 0}[n]{\gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} f\of{\ga_\gn}}
+\]
+where $m ≤ n$ and the coefficients $\gd^{m}_{n,\gn}$ are dependent on $\VectorSymbol{\ga}$ but independent of $f$.
+
+If the $\VectorSymbol{\ga}$ are equally spaced with step $-h$ (\idest, $\ga_\gn = x_0 - \gn h$) then we can write a backward formula as follows:
+\begin{equation}
+f^{(m)}\of{x_0} = \sum{\gn = 0}[n]{\gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} f\of{x_0 - \gn h}} + \BigO\of{h^{n - m + 1}}
+\label{fornbergbackwardpreliminary}
+\end{equation}
+
+In this case equation (3.8) from \cite{Fornberg1988} may be rewritten as follows, restoring $x_0$:
+\begin{align}
+\gd^{m}_{n,\gn} &= \frac{1}{\ga_n - \ga_\gn}\pa{\pa{\ga_n - x_0} \gd^{m}_{n - 1,\gn} - m \gd^{m - 1}_{n - 1,\gn}} \nonumber \\
+&= \frac{1}{\pa{\gn - n} h} \pa{-n h \gd^{m}_{n - 1,\gn} - m \gd^{m - 1}_{n - 1,\gn}} \label{eqndelta}
+\end{align}
+
+It is possible to extract the powers of $h$ from the coefficients by defining, for $\gn < n$:
+\[
+\gl^{m}_{n,\gn} \DefineAs \gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} \pa{-1}^m h^m
+\]
+
+Substituting into equation (\ref{eqndelta}) we obtain:
+\[
+\gl^{m}_{n,\gn} = \frac{1}{n - \gn} \pa{n \gl^{m}_{n - 1,\gn} - m \gl^{m - 1}_{n - 1,\gn}}
+\]
+and we see that $\gl^{m}_{n,\gn}$ is independent from $x_0$ and $h$ and therefore that $\gd^{m}_{n,\gn}$ is independent from $x_0$.
+
+Note that the definition of $\gl^{m}_{n,\gn}$ extends immediately to $n = \gn$, that is to $\gl^{m}_{n,n}$, using equation (3.10) of \cite{Fornberg1988}.
+
+Now knowing that:
+\[
+\gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} = \pa{-1}^m \frac{\gl^{m}_{n,\gn}}{h^m}
+\]
+it is straightforward to rewrite equation (\ref{fornbergbackwardpreliminary}) to obtain equation (\ref{fornbergbackward}), thus demonstrating the lemma.
```

Comment:
<del>Now knowing that:</del><ins>Substituting</ins>
[…]
<del>it is straightforward to rewrite</del><ins>into</ins> equation (\ref{fornbergbackwardpreliminary}) <del>to obtain</del><ins>yields</ins> equation (\ref{fornbergbackward}), thus <del>demonstrating</del>[_Gallicism_]<ins>proving</ins> the lemma.